test/floatapprox.jl

```diff
@@ -6,19 +6,19 @@
 @test 4.000000002 ≉ 4.00300002
 
 # Other tolerance levels
-@test isapprox(4.32, 4.3; rtol=0.1, atol=0.01)
-@test isapprox(1.001, 1.002; rtol=0.001, atol=0.0001)
-@test !isapprox(4.5, 4.9; rtol=0.001, atol=0.001)
+@test 4.32 ≈ 4.3 rtol=0.1 atol=0.01
+@test 1.001 ≈ 1.002 rtol=0.001 atol=0.0001
+@test 4.5 ≉ 4.9 rtol=0.001 atol=0.001
```

Comment:
Ah, I see.  I was hoping it could be used outside of `@test`.  I guess someone could write another macro that would parse it in the same way (without requiring `Base.Test`).